benchmarks/2019.json

```diff
@@ -3401,6 +3455,24 @@
       100
     ]
   },
+  {
+    "measureId": "343",
```

Comment:
This should also be included according to calculation logs from past run and current run

Reply:
Was removed here:
https://github.com/CMSgov/qpp-measures-data/pull/269/files#diff-19cc2b2b0a838f96141210f2b77688c9L3079

Let me ask @saquino0827